### PR TITLE
Add logdna to the dashboard

### PR DESCRIPTION
**Pull request by dfellis**

### CODE REVIEW

File: dashboard/src/pages/index.tsx [hunk 1]
```diff
@@ -30,6 +31,13 @@ export default function App() {
     if (telemetry !== undefined && telemetry === 'on') {
       Sentry.init(config);
       Posthog.init(config);
+      if (config?.logdna?.key) {
+        logdna.init(config.logdna.key);
```
Comment (depombo) [inline dashboard/src/pages/index.tsx:35]: do we need a different one for the server per https://github.com/logdna/logdna-browser#nextjs-and-server-side-rendering or we are good ?

**Review by aguillenv — COMMENTED** [review]
do we need to add explicit log calls?

File: dashboard/src/pages/index.tsx (hunk 1 above)
Comment (dfellis) [reply dashboard/src/pages/index.tsx:35]: Technically we do, but we have only two server endpoints with the `dashboard/src/pages/api/config.ts` and `run.ts` files. In both cases they immediately punt data to the front end so I figured the logging there would handle anything relevant?

**Comment by dfellis** [comment]
> do we need to add explicit log calls?

By [default](https://github.com/logdna/logdna-browser#configuration-options) it intercepts *all* `console.log/warn/error` calls, so we should be fine.

File: dashboard/src/pages/index.tsx (hunk 1 above)
Comment (depombo) [reply dashboard/src/pages/index.tsx:35]: well, `run.ts` could have something off in the Postgres querying? not blocking though, we can add it later if we want

**Review by aguillenv — APPROVED** [review]

**Review by depombo — APPROVED** [review]

**Pull request merged**